### PR TITLE
add outreach supporters gratitudes post

### PR DESCRIPTION
**Pull request by serahkiburu**

### CODE REVIEW

File: _posts/2019/12/2019-12-05-thanking-comms-supporters.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+Here are some of the remarkable efforts that different community members have put in this year that have supported our communications efforts effectively:
+
+#### Fortnightly Summaries, Meaningful Discoveries 
+<br/>
+We have Belinda Weaver and Jonah Duckles to thank for the clever name our newsletter goes by: Carpentry Clippings. 
+
+With over 2,700 subscribers, Carpentries Clippings is one of the main ways many people in our community stay updated and get wind of initiatives that need their attention and expertise. Curating a great newsletter takes a lot of time, and Malvika Sharan has tirelessly championed this work this year. Her fortnightly work board is 20+ tasks long and involves a lot of proactivity, and we are in awe of the quality, precision and timeliness with which Malvika continues to deliver every issue. Thank you so much, Malvika!
```
Comment (kariljordan) [inline _posts/2019/12/2019-12-05-thanking-comms-supporters.md]: Carpentry Clippings (right? not Carpentries Clippings)...first sentence in line 21

File: _posts/2019/12/2019-12-05-thanking-comms-supporters.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+We thank you!
+
+#### Further Together
+
+Every so often in our community discussions, people get to share how they first heard about The Carpentries. More often than not, the answer has been ‘through a podcast episode or keynote/lightning talk / presentation at a conference or webinar or in a discussion on Twitter ’. We would be remiss to not acknowledge how valuable it has been to have so many people fly The Carpentries banner high in different spaces. From pointing people to our curricula, to discussing elements we can improve about our processes and inviting people to join in and collaborate in our community, you have represented The Carpentries favourably and impeccably, and we celebrate your enthusiasm and willingness to do so time and again.
```
Comment (kariljordan) [inline _posts/2019/12/2019-12-05-thanking-comms-supporters.md]: keynote/lightning talk / presentation 

I think it's missing a space, so should be:

keynote / lightning talk / presentation

or

keynote/lightning talk/presentation

**Review by kariljordan — COMMENTED** [review]
This post made me smile. Beautifully written, and I'm so appreciative of the way you're shining a light on our community members.

File: _posts/2019/12/2019-12-05-thanking-comms-supporters.md (hunk 1 above)
Comment (serahkiburu) [reply _posts/2019/12/2019-12-05-thanking-comms-supporters.md]: good spot! changing it

**Comment by serahkiburu** [comment]
thanks for reviewing @kariljordan, made edits

**Pull request merged**